docs/contributing/doc_guide.rst

```diff
@@ -497,6 +497,30 @@ example, ``:cite:p:`wilson:2014``` will show up as :cite:p:`wilson:2014`,
 ``:cite:p:`wilson:2014, wilson:2017``` will show up as
 :cite:p:`wilson:2014, wilson:2017`.
 
+.. _api-static:
+
+Creating a documentation stub file for a new module
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the narrative documentation does not index a subpackage (a
+directory) or module (a :file:`.py` file) with ``automodule``,
+``automodapi``, or the like, then it is required to create a stub file
+for that particular subpackage or module in |docs/api_static|_. For
+example, the stub file for `plasmapy.particles.atomic` is placed at
+:file:`docs/api_static/plasmapy.particles.atomic.rst` and its contents
+look like:
+
+  .. code-block:: rst
+
+     :orphan:
+
+     `plasmapy.particles.atomic`
+     ===========================
+
+     .. currentmodule:: plasmapy.particles.atomic
+
+     .. automodapi::  plasmapy.particles.atomic
+
```

Comment:
Should I add a note here that if a new folder (i.e. a subpackage) is created with a `.py` file in it (i.e. a module), that a stub file will need to be created for both of them?  Need to double check this...